libsoftwarecontainer/src/container.cpp

```diff
@@ -640,40 +644,74 @@ bool Container::bindMountCore(const std::string &pathInHost,
     std::string filePart = baseName(pathInContainer);
     std::string tempDirInContainer = buildPath(gatewaysDirInContainer(), filePart);
 
-    // The same paths on the host
-    std::string pathInContainerOnHost = buildPath(rootFS(), pathInContainer);
+    pid_t pid = INVALID_PID;
+    int status = -1;
+
+    // Utility lambda function to create parent paths of a given path
+    auto createParentDirectories = [this] (std::string path) {
```

Comment:
We can't use the method from FileToolKitWithUndo, because that uses logging, and until the DLT is fixed to work well with pid namespaces we can't log with that inside containers.